Exiled.Loader/Loader.cs

```diff
@@ -120,6 +120,9 @@ public Loader()
         /// </summary>
         public static void LoadPlugins()
         {
+            // Required to remove the old ExiledUpdater than now integrated inside Exiled.Loader
+            File.Delete($"{Paths.Plugins}/Exiled.Updater.dll");
```

Comment:
use `Path.Combine(Paths.Plugins, "Exiled.Updater.dll")`

separating symbol is different on Win and Unix

Reply:
Oh okay yes